man/man8/compactsnoop.8

```diff
@@ -0,0 +1,179 @@
+.TH compactsnoop 8  "2019-11-1" "USER COMMANDS"
+.SH NAME
+compactstall \- Trace compact zone events. Uses Linux eBPF/bcc.
+.SH SYNOPSIS
+.B compactsnoop.py [\-h] [\-T] [\-p PID] [\-d DURATION] [\-K] [\-e]
+.SH DESCRIPTION
+compactsnoop traces the compact zone events, showing which processes are
+allocing pages with memory compaction. This can be useful for discovering
+when compact_stall (/proc/vmstat) continues to increase, whether it is
+caused by some critical processes or not.
+
+This works by tracing the compact zone events using raw_tracepoints and one
+kretprobe.
+
+For the Centos 7.6 (3.10.x kernel), see the version under tools/old, which 
+uses an older memory compaction mechanism.
```

Comment:
I am not 100% sure whether we want to support the old version or not. Are you aware of any demand for old kernel support?

Reply:
As many companies in mainland China still use RHEL/CentOS 7 as the production environment, I can't see any signs of upgrading to RHEL/CentOS 8 or ubuntu 18.04 in the short term, so I added this. I hope they can experience the power of BCC in their production environment. But if the BCC community does not want to support the old kernel, I'm fine to remove it which may also be a driving force for upgrading the kernel. 3.10.x is too old. :(

Reply:
okay, then we can have a version under old/ for the benefit of existing users. thanks!